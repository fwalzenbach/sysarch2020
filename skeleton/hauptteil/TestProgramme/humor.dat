0x24010539
0x2422002A
0x24080045
0x240901A4
0x240A0003
0x00481823
0x00692023
0x00AA0019
0x00003012
0x00003810