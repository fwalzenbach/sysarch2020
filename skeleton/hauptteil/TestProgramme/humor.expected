@1 0x00000539
@2 0x00000563
@3 0x0000051e
@4 0x0000037a
@6 0x00000a6e
@7 0x00000000
@8 0x00000045
@9 0x000001a4
@a 0x00000003